.taskcluster.yml

```diff
@@ -41,6 +41,8 @@ tasks:
             - "-cx"
             - "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py &&
                rustup component add clippy rustfmt &&
+               rustup toolchain install nightly &&
+               curl -L https://github.com/est31/cargo-udeps/releases/download/v0.1.25/cargo-udeps-v0.1.25-x86_64-unknown-linux-gnu.tar.gz | tar xz -C /usr/local/cargo/bin --strip-components 2 &&
```

Comment:
What is this needed for?

Reply:
To install `cargo udeps` fastly using the binary

Reply:
Let's use cargo install instead, it's more future-proof.

Reply:
The command took a while to run and slowed down the CI a lot. If we use the binary, we can pin a specific version avoiding CI breaking in the future also

Reply:
We can pin to a specific version anyway, by running `cargo install udeps --version 0.1.25` or simply adding it to the dev-dependencies in Cargo.toml (so we also benefit from Dependabot).
If it is slowing down the task too much we can keep it this way though.

Reply:
True about the `cargo install` command, just clarifying that we can pin to a specific version using the binary too